cypress/support/utils/dateTools.js

```diff
@@ -1,5 +1,6 @@
 import moment from 'moment';
 import getRandomStringCode from './genereteTextCode';
+
```

Comment:
Need here that?)

Reply:
there should be a new line after the import statements